docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -500,7 +500,7 @@ The docker-compose up and docker run commands (or running and debugging the cont
 
 ## Step 6. Test your Docker application using your local Docker host
 
-This step will vary depending on what your application is doing. In a simple .NET Core Web application that is deployed as a single container or service, you can access the service by opening a browser on the Docker host and navigating to that site, as shown in Figure 5-13. (If the configuration in the Dockerfile maps the container to a port on the host that is anything other than 80, include the host port in the URL.)
+This step will vary depending on what your application is doing. In a simple .NET Web application that is deployed as a single container or service, you can access the service by opening a browser on the Docker host and navigating to that site, as shown in Figure 5-13. (If the configuration in the Dockerfile maps the container to a port on the host that is anything other than 80, include the host port in the URL.)
```

Comment:
Not sure about this one. For web apps, the suffix "Core" wasn't dropped (i.e, ASP.NET Core 5.0, not ASP.NET 5.0). It may be clearer to say "In a simple ASP.NET Core Web application" instead.

Reply:
@Youssef1313 This is calling the proper name though, this is accurate - they are all .NET web apps.